Loenn/effects/cloudscape.lua

```diff
@@ -48,13 +48,17 @@ cloudscape.fieldInformation = {
     rotationExponent = {
         minimumValue = 0.0,
         maximumValue = 2.0
+    },
+    alpha = {
+        minimumValue = 0.0,
+        maximumValue = 1.0
     }
 }
 
 cloudscape.defaultData = {
     seed = "",
-    colors = "6d8ada,aea0c1,d9cbbc",
-    bgColor = "4f9af7",
+    colors = "6d8adaff,aea0c1ff,d9cbbcff",
+    bgColor = "4f9af7ff",
```

Comment:
you've changed the defaults here but not the fallbacks in `Cloudscape.cs`; ideally those should match